public/css/planets.css

```diff
@@ -66,12 +71,154 @@ header {
     height: 35vw;
     border: 1px solid white;
     margin: 1.5vw;
+	
+	background-image:url("/static/images/Background1-18.png");
```

Comment:
Also, there is four different starry images, and these four should be randomly picked as a background in all divs. See Google drive for both images and description.